WebARonARKit/WebARonARKit/Shaders.metal

```diff
@@ -27,152 +28,167 @@ typedef struct {
     float2 texCoord[[attribute(kVertexAttributeTexcoord)]];
 } ImageVertex;
 
-
 typedef struct {
     float4 position[[position]];
     float2 texCoord;
 } ImageColorInOut;
 
-
 // Captured image vertex function
-vertex ImageColorInOut capturedImageVertexTransform(ImageVertex in[[stage_in]])
-{
+vertex ImageColorInOut
+capturedImageVertexTransform(ImageVertex in[[stage_in]]) {
     ImageColorInOut out;
-
+    
     // Pass through the image vertex's position
     out.position = float4(in.position, 0.0, 1.0);
-
+    
     // Pass through the texture coordinate
     out.texCoord = in.texCoord;
-
+    
     return out;
 }
 
 // Captured image fragment function
-fragment float4 capturedImageFragmentShader(ImageColorInOut in[[stage_in]],
-                                            texture2d<float, access::sample> capturedImageTextureY[[texture(kTextureIndexY)]],
-                                            texture2d<float, access::sample> capturedImageTextureCbCr[[texture(kTextureIndexCbCr)]])
-{
-
-    constexpr sampler colorSampler(mip_filter::linear,
-                                   mag_filter::linear,
+fragment float4 capturedImageFragmentShader(
+                                            ImageColorInOut in[[stage_in]],
+                                            texture2d<float, access::sample>
+                                            capturedImageTextureY[[texture(kTextureIndexY)]],
+                                            texture2d<float, access::sample>
+                                            capturedImageTextureCbCr[[texture(kTextureIndexCbCr)]]) {
+    constexpr sampler colorSampler(mip_filter::linear, mag_filter::linear,
                                    min_filter::linear);
-
-    const float4x4 ycbcrToRGBTransform = float4x4(
-        float4(+1.0000f, +1.0000f, +1.0000f, +0.0000f),
-        float4(+0.0000f, -0.3441f, +1.7720f, +0.0000f),
-        float4(+1.4020f, -0.7141f, +0.0000f, +0.0000f),
-        float4(-0.7010f, +0.5291f, -0.8860f, +1.0000f));
-
-    // Sample Y and CbCr textures to get the YCbCr color at the given texture coordinate
-    float4 ycbcr = float4(capturedImageTextureY.sample(colorSampler, in.texCoord).r,
+    
+    const float4x4 ycbcrToRGBTransform =
+    float4x4(float4(+1.0000f, +1.0000f, +1.0000f, +0.0000f),
+             float4(+0.0000f, -0.3441f, +1.7720f, +0.0000f),
+             float4(+1.4020f, -0.7141f, +0.0000f, +0.0000f),
+             float4(-0.7010f, +0.5291f, -0.8860f, +1.0000f));
+    
+    // Sample Y and CbCr textures to get the YCbCr color at the given texture
+    // coordinate
+    float4 ycbcr = float4(
+                          capturedImageTextureY.sample(colorSampler, in.texCoord).r,
```

Comment:
it is weird to have this start on the next line in the same spot it could have started on the previous line. Move to previous line